tests/mock_network/node_ageing.rs

```diff
@@ -256,9 +258,6 @@ fn section_churn(
     assert!(min_section_size < max_section_size);
 
     let mut rng = network.new_rng();
-    let next_relocation_dst = unwrap!(nodes_with_prefix(nodes, prefix).next())
```

Comment:
How does that bit now work without that?

Reply:
The overrides are now shared, so they would apply to any new node too.